synapse/models/inet.py

```diff
@@ -692,45 +824,44 @@ def getBaseModels():
 
                     ('url', {'ptype': 'inet:url'}),
                     ('webpage', {'ptype': 'inet:url'}),
                     ('avatar', {'ptype': 'file:bytes'}),
 
-                    ('tagline', {'ptype': 'str:txt', 'doc': 'A netuser status/tag line text'}),
-                    ('occupation', {'ptype': 'str:txt', 'doc': 'A netuser self declared occupation'}),
+                    ('tagline', {'ptype': 'str:txt', 'doc': 'A web account status/tag line text'}),
+                    ('occupation', {'ptype': 'str:txt', 'doc': 'A web account self declared occupation'}),
                     # ('gender',{'ptype':'inet:fqdn','ro':1}),
 
                     ('name', {'ptype': 'inet:user'}),
                     ('realname', {'ptype': 'ps:name'}),
                     ('email', {'ptype': 'inet:email'}),
                     ('phone', {'ptype': 'tel:phone'}),
-                    ('signup', {'ptype': 'time', 'doc': 'The time the netuser account was registered'}),
-                    ('signup:ipv4',
-                     {'ptype': 'inet:ipv4', 'doc': 'The original ipv4 address used to sign up for the account'}),
-                    ('passwd', {'ptype': 'inet:passwd', 'doc': 'The current passwd for the netuser account'}),
+                    ('signup', {'ptype': 'time', 'doc': 'The time the web account was registered'}),
+                    ('signup:ipv4', {'ptype': 'inet:ipv4', 'doc': 'The original ipv4 address used to sign up for the account'}),
+                    ('passwd', {'ptype': 'inet:passwd', 'doc': 'The current passwd for the web account'}),
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 ]),
 
                 ('inet:web:logon', {'ptype': 'inet:web:logon'}, [
-                    ('netuser', {'ptype': 'inet:netuser', 'doc': 'The netuser associated with the logon event.', }),
-                    ('netuser:site', {'ptype': 'inet:fqdn', }),
-                    ('netuser:user', {'ptype': 'inet:user', }),
-                    ('time', {'ptype': 'time', 'doc': 'The time the netuser logged into the service', }),
+                    ('acct', {'ptype': 'inet:web:acct', 'doc': 'The account associated with the logon event.', }),
+                    ('acct:site', {'ptype': 'inet:fqdn', }),
+                    ('acct:user', {'ptype': 'inet:user', }),
+                    ('time', {'ptype': 'time', 'doc': 'The time the account logged into the service', }),
                     ('ipv4', {'ptype': 'inet:ipv4', 'doc': 'The source IPv4 address of the logon.'}),
                     ('ipv6', {'ptype': 'inet:ipv6', 'doc': 'The source IPv6 address of the logon.'}),
-                    ('logout', {'ptype': 'time', 'doc': 'The time the netuser logged out of the service.'})
+                    ('logout', {'ptype': 'time', 'doc': 'The time the account logged out of the service.'})
```

Comment:
We've talked about the difficulty of identifying complete "sessions" (paired login / logout events). I'm ok leaving this as an optional secondary prop (on the off chance we have that data); however, we could also capture logouts as generic inet:web:action forms. Thoughts?

Reply:
We can discuss this, but it won't be changed in this PR at least.

Reply:
I'll add tracking for this but not change it as part of this PR.

Reply:
This will not be removed per discussion.